src/NLog/Targets/Wrappers/FallbackGroupTarget.cs

```diff
@@ -127,11 +130,13 @@ protected override void Write(AsyncLogEventInfo logEvent)
                         // success
                         if (ReturnToFirstOnSuccess)
                         {
+#pragma warning disable S1066 // Collapsible "if" statements should be merged
 #if !SILVERLIGHT || WINDOWS_PHONEs
```

Comment:
Funny with `WINDOWS_PHONEs`

Reply:
yes good know, also nice it just compiles.

Reply:
not sure what to do about this (anyway, not in the PR)

What do you think?

Reply:
Created #3320